releaseNotes/AzureSpringCloudAppConfigurationConfig.md

```diff
@@ -6,6 +6,15 @@
 
 [Source code web][source_code_web] | [Package (Maven) web][package_web] | [Product documentation][docs]
 
+## 2.5.0 - March 29, 2022
+
+* Added configuration for periodically reloading all configurations. Can be used for forcing Key Vault references can be updated. `spring.cloud.azure.appconfiguration.refresh-interval= 1d`
```

Comment:
Add a refresh interval `spring.cloud.azure.appconfiguration.refresh-interval= 1d` that can be used to periodically reload all the key-value pairs in a configuration store. This can also be used to force update Key Vault References.